libs/spark/src/Unstable_SectionMessageTitle/Unstable_SectionMessageTitle.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import { render } from '@testing-library/react';
+import Unstable_SectionMessageTitle from './Unstable_SectionMessageTitle';
+
+describe('Unstable_SectionMessageTitle', () => {
+  it('Can render without ThemeProvider', () => {
+    const { baseElement } = render(
+      <Unstable_SectionMessageTitle title="Test">
+        Testy Test
+      </Unstable_SectionMessageTitle>
```

Comment:
```suggestion
      <Unstable_SectionMessageTitle>
        Title
      </Unstable_SectionMessageTitle>
```
Mistaken carryover of `Unstable_SectionMessage` render format.